src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -768,30 +752,22 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // - Set the font size of the terminal control.
     // Arguments:
     // - fontSize: The size of the font.
-    void ControlCore::_setFontSize(int fontSize)
+    // Return Value:
+    // - Returns true if you need to call _refreshSizeUnderLock().
+    bool ControlCore::_setFontSizeUnderLock(int fontSize)

```

Comment:
Oh. So it's not doing it "under lock". It's saying that you should call this only if you're under a lock.

Reply:
Yeah, there's another function with a similar name pattern.